content/en/news/support/announcing-1.8-eol/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Support for Istio 1.8 ends on May 12th, 2021
+subtitle: Support Announcement
+description: Upcoming Istio 1.8 end of life announcement.
+publishdate: 2021-04-12
+aliases:
+    - /news/2021/announcing-1.8-eol
+---
+
+According to Istio's [support policy](/docs/releases/supported-releases#supported-releases/), LTS releases like 1.8 are supported for three months after the next LTS release. Since [1.9 was released on February 9th](/news/releases/1.8.x/announcing-1.9/), support for 1.8 will end on May 12th, 2021.
```

Comment:
@duderino @myidpt 

As @howardjohn suggested, I extended the support window by 3 days to make 1.8 also include the 5/11 fix. Does this make sense? The wording seems a bit weird though, I guess we'd better make it clear there is a three days extension?

Reply:
That sounds good to me.